x/programs/rust/wasmlanche_sdk/src/store/mod.rs

```diff
@@ -1,40 +1,48 @@
 use crate::errors::StorageError;
-use crate::host::host_program_invoke;
+use crate::host::program_invoke;
 use crate::host::{get_bytes, get_bytes_len, store_bytes};
 use crate::types::Argument;
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 use serde_bare::{from_slice, to_vec};
 use std::str;
 
-pub enum StoreResult<'a> {
-    Ok(&'a Context),
+#[allow(clippy::module_name_repetitions)]
```

Comment:
This is another stutter, but I think it's fine to ignore here since causing a collision with `std::result::Result` would be a pain.